api/graphql/schema.graphql

```diff
@@ -189,6 +189,29 @@ type PoolCandidateSearchRequest {
     adminNotes: String @rename(attribute: "admin_notes")
 }
 
+enum SkillCategory {
+    TECHNICAL,
+    BEHAVIOURAL
+}
+
+type SkillFamily {
+    id: ID!
+    key: KeyString!
+    name: LocalizedString!
+    description: LocalizedString!
+    skills: [Skill] @hasMany
```

Comment:
Shouldn't this and line 212 have the directive @belongsToMany instead?

Reply:
Can't get anything past you!  Great catch. 😀